sdk/formrecognizer/Azure.AI.FormRecognizer/src/RecognizeContentOptions.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.AI.FormRecognizer
+{
+    /// <summary>
+    /// The set of options that can be specified when calling a recognition content method
```

Comment:
nit: I think `a content recognition` method makes more sense, but take my English suggestions with a grain of salt.

Reply:
Or just saying `calling a recognize content method` like you did in the Receipts Options sounds good as well.